C_CPP/Stack/readme.md

```diff
@@ -0,0 +1,7 @@
+# STACK(NEAREST GREATER TO RIGHT IN STACK) #
+----------------------------------------------------------------------------------------
+- Stack foundation problem(earest greater to right or say Next greater element
+- we can solve it using array also but it will have a high time complexity 
+- therfore using stack only 1 loop is suffecient completing IN o(n) time
+- ----------------------------------------------------------------------------------------
+(https://github.com/keshav12122000/winter-of-contributing/blob/3ba65ee82f48544c34fc379762a169fb97a6dfd5/C_CPP/Stack/NGR.cpp)
```

Comment:
Link will break, if you change the fork later ... So remove it
Use relative link